src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -2586,6 +2587,7 @@ namespace OpenMS
     for (int i = argc - 1; i > 0; --i)
     {
       String arg = argv[i];
+      if (arg.hasPrefix("--")) arg = arg.substr(1); // replace '--' by '-' such that it's found in param_map
```

Comment:
well, help and helphelp are the (unfortunate) exception here, since they were registered before as "-help" (the "-" was part of the parameter name) -- which was not the smartest solution.
So yes, -help on the commandline would also give you the help text. Do you think thats a problem?